kitsune/search/search.py

```diff
@@ -208,6 +210,11 @@ def get_filter(self):
             DSLQ("term", _index=self.get_index()),
             DSLQ("exists", field=f"title.{self.locale}"),
         ]
+
+        if not self.parse_query:
+            # Only exclude archived documents in simple search
+            filters.append(DSLQ("term", is_archived=False))
```

Comment:
Does `is_archived` need to be added to the `WikiDocument` in order to filter on it?

Reply:
Hm, I don't think so.